packages/jellyfish-transaction/src/script/dftx/dftx_loans.ts

```diff
@@ -50,3 +52,34 @@ export class CSetDefaultLoanScheme extends ComposableBuffer<SetDefaultLoanScheme
     ]
   }
 }
+
+/**
+ * DestroyLoanScheme DeFi Transaction
+ */
+export interface DestroyLoanScheme {
+  identifier: string // ------------------| c = VarUInt{1-9 bytes} + c bytes UTF encoded string, Unique identifier of the loan scheme
+  height?: BigNumber // -------------------| 8 bytes unsigned integer, Activation block height
+}
+
+/**
+ * Composable DestroyLoanScheme, C stands for Composable.
+ * Immutable by design, bi-directional fromBuffer, toBuffer deep composer.
+ */
+export class CDestroyLoanScheme extends ComposableBuffer<DestroyLoanScheme> {
+  static OP_CODE = 0x44 // 'D'
+  static OP_NAME = 'OP_DEFI_TX_DESTROY_LOAN_SCHEME'
+
+  composers (dls: DestroyLoanScheme): BufferComposer[] {
+    return [
+      ComposableBuffer.varUIntUtf8BE(() => dls.identifier, v => dls.identifier = v),
+      {
+        fromBuffer: (buffer: SmartBuffer): void => {
+          dls.height = readBigNumberUInt64(buffer)
+        },
+        toBuffer: (buffer: SmartBuffer): void => {
+          writeBigNumberUInt64(dls.height ?? new BigNumber(0x00000000), buffer)
+        }
```

Comment:
https://github.com/DeFiCh/jellyfish/pull/594/files#r699192389

Reply:
Okay, I have changed the code. Now it is bi-directional without height.